back/src/entities/address.entity.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  ManyToOne,
+} from "typeorm"; // Pour définir les entités TypeORM
+import {
+  Field,
+  GraphQLISODateTime,
+  ID,
+  InputType,
+  ObjectType,
+} from "type-graphql"; // Pour définir les Types GraphQL
+
+export type Status = "ACTIVE" | "ARCHIVED";
+
+@ObjectType() //pour TypeGraphQL
+@Entity() // pour TypeORM
+export class AddressEntity {
+  @Field(() => ID) // pour GraphQL
+  @PrimaryGeneratedColumn("uuid") // pour TypeORM
+  id: string;
+
+  @Field()
+  @Column()
+  streetNumber: number;
+
+  @Field()
+  @Column({ type: "varchar", length: 250 })
+  streetName: string;
+
+  @Field()
```

Comment:
https://the-guild.dev/graphql/scalars/docs/scalars/postal-code
@Field(() => PostalCode)